pkg/providers/snow/api/v1beta1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1beta1 contains API Schema definitions for the infrastructure v1beta1 API group
-//+kubebuilder:object:generate=true
-//+groupName=infrastructure.cluster.x-k8s.io
+// +kubebuilder:object:generate=true
```

Comment:
I think you have an autoformatter playing havoc. These lines are read my machines so shouldn't have a space after the `//`.